### PR TITLE
feat: [MP-767] mykiva lending stats

### PR DESCRIPTION
**Pull request by mlibazisi**
Ticket: https://kiva.atlassian.net/browse/MP-767

### CODE REVIEW

File: src/components/MyKiva/MyKivaStats.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<template>
+	<section class="tw-flex tw-overflow-hidden tw-flex-col tw-justify-center tw-items-center tw-px-16 tw-py-8 tw-bg-green-800 tw-max-md:tw-px-5">
```
Comment (emuvente) [inline src/components/MyKiva/MyKivaStats.vue:2]: We need to use colors that are defined in our brand guidelines, and that means not using tailwind color classes like `tw-text-neutral-100` or `tw-bg-green-800`. Instead you can use the color classes we have defined here: . The exact colors you should use are also defined in the Figma design, when you click on the element and look at the "Properties" tab. In this case, I think you should use `tw-bg-eco-green-3` and `tw-text-primary-inverse`.

**Review by emuvente — COMMENTED** [review]
Please try running `npm run lint` on this branch and fix the errors it complains about. You can ignore the warnings.

**Comment by roger-in-kiva** [comment]
Could we use composition api here for consistency with other MyKiva components?

**Comment by roger-in-kiva** [comment]
Here the screenshots with long numbers option.

Desktop:
Screenshot 2024-10-21 at 1 17 38 p m
Screenshot 2024-10-21 at 1 16 54 p m

Ipad:
Screenshot 2024-10-21 at 1 17 45 p m
Screenshot 2024-10-21 at 1 16 12 p m

Mobile:
Screenshot 2024-10-21 at 1 17 55 p m
Screenshot 2024-10-21 at 1 15 46 p m

**Review by dyersituations — APPROVED** [review]

**Pull request merged**